keps/sig-api-machinery/4006-transition-spdy-to-websockets/README.md

```diff
@@ -317,22 +317,6 @@ is redirected to other API endpoints.
 
   - Mitigation: Upgraded connections are disallowed from redirecting.
 
-- Risk: Overloaded Concurrency
```

Comment:
Removed, since this risk was associated with `StreamTranslation`, and it was fixed when the design was updated to `StreamTunneling`.